spec/core/ics-003-connection-semantics/README.md

```diff
@@ -125,6 +125,7 @@ function clientConnectionsPath(clientIdentifier: Identifier): Path {
 function addConnectionToClient(
   clientIdentifier: Identifier,
   connectionIdentifier: Identifier) {
+    abortTransactionUnless(queryClientState(clientIdentifier) !== null)
```

Comment:
IMO this is less clear than if it was a statement in the handlers' body directly ([as we do in ibc-rs](https://github.com/cosmos/ibc-rs/blob/1cbd1d93a87cd1967c943897d199f3274ee84259/crates/ibc/src/core/ics03_connection/handler/conn_open_init.rs#L24)). I also don't like that the check occurs after the `provableStore.set(connectionPath(identifier), connection)` line (e.g. in `ConnOpenTry`). Although I think we state somewhere that changes are reverted when a transaction is aborted, it's clearer to make all checks before state change statements.

Reply:
I see your point, thanks. I think I will defer to @AdityaSripal to decision to make such change. I would  also do it in a different PR that would close #894

Reply:
Don't understand the context here, please fill me in on slack @crodriguezvega

Reply:
Moved the statement to the body of `connOpenInit` and `connOpenTry`.